charts/castai-evictor/templates/secret-tls-cert.yaml

```diff
@@ -0,0 +1,16 @@
+{{- if .Values.trustedCACert }}
```

Comment:
Where is this `trustedCACert` coming from? It should be defined in `values.yaml`.

Reply:
It is an optional value used by a small (currently, 1) subset of customers. Other services doesn't define it in `values.yaml` as well. Should I update those too, or just be consistent and don't define it anywhere?